src/Components/SmartComponents/ClusterList/ClusterListTable.cy.js

```diff
@@ -15,14 +14,17 @@ import {
   itIsNotSorted,
   itIsSortedBy,
 } from '../../../../cypress/utils/table';
+import { MemoryRouter, Route, Routes } from 'react-router-dom';
 
 const mountComponent = () => {
   mount(
-    <Router>
+    <MemoryRouter>
```

Comment:
I believe you can remove this router completely.

Reply:
I wasn't able to get it working without a router, propably because chrome is not running in these cypress tests.

Reply:
This is true, the components here are rendered without chroming => they will miss router

Reply:
Oh, i missed that this is for cypress tests. :D my bad. ignore me.